### PR TITLE
Run retries helm config

### PR DESCRIPTION
**Pull request by johannkm**

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #8369** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/8369" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/8369?utm_source=stack-comment).

File: helm/dagster/values.yaml [hunk 1]
```diff
@@ -999,6 +999,10 @@ dagsterDaemon:
     # run launcher supports resuming runs, otherwise defaults to 0.
     maxResumeRunAttempts: ~
 
+  runRetries:
```
Comment (johannkm) [inline helm/dagster/values.yaml:1002]: will link to docs once https://app.graphite.dev/github/pr/dagster-io/dagster/8367 lands

File: helm/dagster/schema/schema/charts/dagster/subschema/daemon.py [hunk 2]
```diff
@@ -81,6 +81,7 @@ class Daemon(BaseModel):
     startupProbe: kubernetes.StartupProbe
     annotations: kubernetes.Annotations
     runMonitoring: Dict[str, Any]
+    runRetries: Dict[str, Any]
```
Comment (rexledesma) [inline helm/dagster/schema/schema/charts/dagster/subschema/daemon.py:84]: Could make this into a typed object - just need to assert it against the config type. See how we did it with the compute logs manager for example: https://sourcegraph.com/github.com/dagster-io/dagster/-/blob/helm/dagster/schema/schema_tests/test_instance.py?L508%3A93=

**Review by rexledesma — APPROVED** [review]
You can have finer types here, up to you

File: helm/dagster/schema/schema/charts/dagster/subschema/daemon.py (hunk 2 above)
Comment (johannkm) [reply helm/dagster/schema/schema/charts/dagster/subschema/daemon.py:84]: Ah nice, will follow up

**Pull request merged**